cfme/infrastructure/virtual_machines.py

```diff
@@ -684,6 +685,28 @@ def active(self):
             else:
                 return has_child(view.tree, '{} (Active)'.format(title), root_item)
 
+        @property
+        def size(self):
+            """
+            Check the shapshot size in the UI. So far available only in RHV and CFME > "5.11"
+
+            :returns the size of the snapshot
+            """
+            from cfme.infrastructure.provider.rhevm import RHEVMProvider
+            if not self.parent_vm.provider.one_of(RHEVMProvider):
+                raise Exception("Provider is not RHV, this feature is not available")
+            if self.parent_vm.appliance.version < "5.11":
+                raise Exception("This feature is available only starting in CFME 5.11")
+            title = getattr(self, self.parent_vm.provider.SNAPSHOT_TITLE)
```

Comment:
Do we want to capture `AttributeError`? Also, do we want to check `title` for being not `None`?

Reply:
Thanks. I don't think it's necessary here. If for some reason `getattr(self, self.parent_vm.provider.SNAPSHOT_TITLE)` fails with AttributeError or returns an empty string, the error message will be clear and will probably indicate a UI change. Please correct me if I'm wrong.

Reply:
Thanks @lina-nikiforova that makes sense. :+1: